app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -2,53 +2,26 @@ package com.kickstarter.libs
 
 import com.kickstarter.libs.KoalaContext.*
 import com.kickstarter.libs.KoalaEvent.ProjectAction
-import com.kickstarter.libs.utils.BooleanUtils
-import com.kickstarter.libs.utils.ExperimentData
 import com.kickstarter.libs.utils.AnalyticEventsUtils
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.ADD_ONS_CONTINUE
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.PLEDGE_INITIATE
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.PLEDGE_SUBMIT
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.REWARD_CONTINUE
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.DISCOVER_FILTER
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.DISCOVER_SORT
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.WATCH_PROJECT
-import com.kickstarter.libs.utils.EventContextValues.ContextTypeName.WATCH
-import com.kickstarter.libs.utils.EventContextValues.ContextTypeName.UNWATCH
-import com.kickstarter.libs.utils.EventContextValues.ContextTypeName.CREDIT_CARD
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.ADD_ONS
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.CHECKOUT
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.UPDATE_PLEDGE
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.REWARDS
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.PROJECT
-import com.kickstarter.libs.utils.EventContextValues.PageViewedContextName.THANKS
-import com.kickstarter.libs.utils.EventContextValues.CtaContextName.SEARCH
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.PWL
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.SUBCATEGORY_NAME
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.CATEGORY_NAME
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.SOCIAL
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.RECOMMENDED
-import com.kickstarter.libs.utils.EventContextValues.DiscoveryContextType.WATCHED
-import com.kickstarter.libs.utils.EventName.CTA_CLICKED
-import com.kickstarter.libs.utils.EventName.PAGE_VIEWED
-import com.kickstarter.libs.utils.EventName.CARD_CLICKED
-import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_CTA
-import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_TYPE
-import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_PAGE
-import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_SECTION
-import com.kickstarter.libs.utils.ContextPropertyKeyName.CONTEXT_LOCATION
+import com.kickstarter.libs.utils.BooleanUtils
+import com.kickstarter.libs.utils.ContextPropertyKeyName.*
```

Comment:
Can we revert these back to not use the wild card imports? Thanks!

Reply:
sure. thanks